includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -1745,6 +1745,9 @@ public function wrap_modal_in_lightbox( $args = [] ) {
 		$open_xpaths  = isset( $args['open_button_xpath'] ) ? $args['open_button_xpath'] : [];
 		$close_xpaths = isset( $args['close_button_xpath'] ) ? $args['close_button_xpath'] : [];
 
+		// Ensure anchor links also close the modal the same as the the Close Menu button.
+		$close_xpaths[] = sprintf( "//*[ @id = '{$modal_id}' ]//a[ @href and contains( @href, '#' ) ]" );
```

Comment:
A really minor nitpick:

The array of `$close_xpaths` actually come from `::add_twentytwenty_modals()`, so I'd expect to see it in that method since it's specific to the Twenty Twenty theme:

```diff
diff --git a/includes/sanitizers/class-amp-core-theme-sanitizer.php b/includes/sanitizers/class-amp-core-theme-sanitizer.php
--- a/includes/sanitizers/class-amp-core-theme-sanitizer.php	(revision 6b178cf6d22b945aa9a2c5289c997e10ad899e21)
+++ b/includes/sanitizers/class-amp-core-theme-sanitizer.php	(date 1608588608663)
@@ -1745,9 +1745,6 @@
 		$open_xpaths  = isset( $args['open_button_xpath'] ) ? $args['open_button_xpath'] : [];
 		$close_xpaths = isset( $args['close_button_xpath'] ) ? $args['close_button_xpath'] : [];
 
-		// Ensure anchor links also close the modal the same as the the Close Menu button.
-		$close_xpaths[] = sprintf( "//*[ @id = '{$modal_id}' ]//a[ @href and contains( @href, '#' ) ]" );
-
 		$modal_actions = [
 			"{$modal_id}.open"  => $open_xpaths,
 			// Although we add the 'show-modal' class here, we don't remove it again, as it will
@@ -1888,6 +1885,9 @@
 			// the actionable elements in the modal. Therefore, the lightbox represents the "background".
 			$close_button_xpaths[] = "//*[ @id = '{$modal_id}' ]";
 
+			// Ensure anchor links also close the modal the same as the the Close Menu button.
+			$close_button_xpaths[] = sprintf( "//*[ @id = '{$modal_id}' ]//a[ @href and contains( @href, '#' ) ]" );
+
 			// Then, add the inner element of the lightbox as an open button xpath.
 			// This is done to prevent the above close action from closing the modal when an inner element is clicked.
 			// Workaround found here: https://stackoverflow.com/a/45971501 .
```

Reply:
Cool. Would you amend the PR with a commit making that change?

Reply:
Sure. Done in [`5316e56` (#5720)](https://github.com/ampproject/amp-wp/pull/5720/commits/5316e563fe40376113a4d12274b92fbffe83d124).